SDWebImage/SDWebImageDownloaderOperation.m

```diff
@@ -496,6 +497,16 @@ - (void)callCompletionBlocksWithImage:(nullable UIImage *)image
                                 error:(nullable NSError *)error
                              finished:(BOOL)finished {
     NSArray<id> *completionBlocks = [self callbacksForKey:kCompletedCallbackKey];
+    
+    if (!completionBlocks.count) {
+        return;
+    }
+    
+    // CallbackBlocks only should be called once
+    LOCK(self.callbacksLock);
+    [self.callbackBlocks removeAllObjects];
+    UNLOCK(self.callbacksLock);
```

Comment:
@zhongwuzw  When code execute `[self.callbackBlocks removeAllObjects];`, It will affect the normal picture `SDWebImageDownloaderProgressiveLoad` logic. Right？

Reply:
@kinarob Thanks, I already forgot we have progressive load.